package-overrides/npm/angular2@2.0.0-beta.8.json

```diff
@@ -0,0 +1,14 @@
+{
+  "dependencies": {
+    "zone.js": "npm:zone.js@0.6.2",
```

Comment:
Its not a big deal to just leave these out altogether, all we really need is:
https://github.com/jspm/registry/blob/angular/package-overrides/npm/angular2%402.0.0-beta.8.json#L13 and the `empty` flag